python/ray/actor.py

```diff
@@ -395,7 +397,10 @@ def _remote(self,
         if is_direct_call is None:
             is_direct_call = bool(os.environ.get("RAY_FORCE_DIRECT"))
         if max_concurrency is None:
-            max_concurrency = 1
+            if is_asyncio:
+                max_concurrency = 500
```

Comment:
How about 100 by default? Some thread stacks could be bigger and cause crashes at 500.